js/apps/system/_admin/aardvark/APP/react/src/views/databases/useFetchInitialValues.ts

```diff
@@ -0,0 +1,20 @@
+import useSWR from "swr";
+import { getAdminRouteForCurrentDB } from "../../utils/arangoClient";
+
+export const useFetchInitialValues = () => {
+  const { data } = useSWR(`/databaseDefaults`, async () => {
+    const route = getAdminRouteForCurrentDB();
+    const res = await route.get(`server/databaseDefaults`);
```

Comment:
we should probably handle the error case somewhere, maybe wrap this in a `try..catch` & show a message to the user? Because if `initialValues` are not present, our modal will not show up at all

Reply:
That's not correct, we are exposing useSWR's error in [some](https://github.com/arangodb/arangodb/blob/devel/js/apps/system/_admin/aardvark/APP/react/src/views/graphV2/useFetchGraphData.tsx#L32) [cases](https://github.com/arangodb/arangodb/blob/devel/js/apps/system/_admin/aardvark/APP/react/src/views/views/editView/useFetchViewProperties.tsx#L12). 
In places we are not, we should.

Let's do it for whatever we add now? @pluma4345